api/client/client.go

```diff
@@ -1319,6 +1319,54 @@ func (c *Client) DeleteAllNodes(ctx context.Context, namespace string) error {
 	return trail.FromGRPC(err)
 }
 
+// StreamSessionEvents streams audit events from a given session recording.
+func (c *Client) StreamSessionEvents(ctx context.Context, sessionID string, startIndex int) (chan events.AuditEvent, chan error) {
```

Comment:
Noticed that events.AuditEvent.GetIndex() is int64, while here it is normal int.

Reply:
Thanks, will update.

Reply:
Fixed